cpp/src/io/json/json_column.cu

```diff
@@ -594,8 +594,7 @@ void make_device_json_column(device_span<SymbolT const> input,
     col.validity =
       cudf::detail::create_null_mask(col.num_rows, cudf::mask_state::ALL_NULL, stream, mr);
     col.type = json_col_t::StringColumn;
-    col.child_columns.clear();  // their references should be deleted too.
-    col.column_order.clear();
+    // destroy references of all child columns after
```

Comment:
Sorry, I'm not understanding this comment.
Destroy references of all child columns after what?

Reply:
This is my only comment about this PR as well. Happy to approve once this is fixed.

Reply:
'after this step'.